client/src/utils/userUtils.js

```diff
@@ -2,19 +2,45 @@ const baseUrl = import.meta.env.VITE_API_URL || 'http://localhost:3000'
 
 import axios from 'axios'
 
+
 // Tracks if landing page submission is login or register
 export const OPTIONS = {
     LOGIN : "login",
     REGISTER : "register"
 }
 
-export const login = (username, password) => {
-    console.log("Login:", username, password)
-    // stuff will happen here
+// Handle login/register submission
+export const handleLoginRegister = (formData, submissionType, setIsSuccessful) => {
+    const username = formData.get("username"), password = formData.get("password")
+
+    submissionType == OPTIONS.LOGIN      && login(username, password, setIsSuccessful)
+    submissionType == OPTIONS.REGISTER   && register(username, password, setIsSuccessful)
+
+    if (submissionType != OPTIONS.LOGIN && submissionType != OPTIONS.REGISTER) {
+        console.error("Invalid type for handleLoginRegister")
+    }
+}
+
+export const login = (username, password, setIsSuccessful) => {
+    axios.post(`${baseUrl}/users/login`, {username, password})
+        .then(res => {
+            console.log("Login successful!")
```

Comment:
what's the reason behind adding logs? if this was just to debug while implementing this, we should remove them to keep the code clean (usually in prod code, we avoid leaving logs we used for debugging)